Makefile

```diff
@@ -57,8 +57,8 @@ LIB_SRC = \
 				${addprefix Channel/, \
 					Channel \
 				} \
-				${addprefix IrcMessage/, \
-					IrcMessage \
+				${addprefix Message/, \
+					Message \
```

Comment:
Please separate it as core.cpp and methods.cpp instead of Message.cpp and core.cpp